frontend/src/app/billing/components/BillingAlerts.tsx

```
@@ -0,0 +1,62 @@
+import { useSearchParams } from 'next/navigation';
+import React from 'react';
+
+type Props = {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)